__tests__/app.test.js

```diff
@@ -356,5 +357,33 @@ describe("GET /api/articles/:article_id", () => {
   });
   
 
+  describe("GET can sort by different fields", () => {
+    test("200: /api/articles?sort_by=title", () => {
+      return request(app)
+        .get("/api/articles?sort_by=title")
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          expect(articles).toBeSortedBy("created_at", {ascending: true ,coerce: true,});
+        });
+    });
```

Comment:
This is not a valid test. Your query on line 363 is: `/api/articles?sort_by=title`, but your assertion is expected it to be sorted by "created_at" and the second argument object is also not being used correctly here. The key ascending is not expected and is ignored. 

I would recommend re-writing this test and revisiting the docs for jest-sorted